deployment/deploy/2120-deploy-DummyFallbackOracle.js

```diff
@@ -0,0 +1,20 @@
+const path = require("path");
+const col = require("cli-color");
+//const deploymentName = getContractNameFromScriptFileName(path.basename(__filename));
+const func = async function (hre) {
+    const {
+        deployments: { deploy, log },
+        getNamedAccounts,
+    } = hre;
+    const { deployer } = await getNamedAccounts(); //await ethers.getSigners();
+    log(col.bgYellow("Deploying DummyFallbackOracle..."));
+    await deploy("FallbackOracle", {
+        contract: "DummyFallbackOracle",
+        from: deployer,
+        args: [],
+        log: true,
+        skipIfAlreadyDeployed: true,
```

Comment:
why is skipIfAlreadyDeployed parameter set to true?

Reply:
I think, it will just skip the deployment if the bytecode is the same, otherwise it will do the redeployment

Reply:
redeployment upon the bytecode change is the default behaviour.   
but this parameter prevents from re-deployment in any case.  
e.g. an upgradeable proxy - if it is never supposed to be re-deployed after the initial deployment with the `skipIfAlreadyDeployed`  it will be skipped every time even if the bytecode is changed.